ts/components/core/selection/CheckBox.tsx

```diff
@@ -19,19 +17,10 @@ type OwnProps = Props &
  * @param props
  * @constructor
  */
-export const CheckBox: React.FunctionComponent<OwnProps> = props => {
-  const [toggleValue, setToggleValue] = useState(props.checked ?? false);
-
-  return (
-    <RawCheckBox
-      checked={toggleValue}
-      disabled={props.disabled}
-      onPress={() => {
-        setToggleValue(!toggleValue);
-        if (props.onValueChange !== undefined) {
-          props.onValueChange(!toggleValue);
-        }
-      }}
-    />
-  );
-};
```

Comment:
With these changes the `CheckBox` ( a component with the automatic state management that uses a `RawCheckBox`) will became identical to a `RawCheckBox`, without differences. 
I think this component should not change and if we need a stateless checkbox, we can use the raw directly.